api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -352,34 +345,76 @@ public DomainValuePair apply(DataSetValues dataSetValue) {
       String workspaceNamespace, String workspaceId, DataSetExportRequest dataSetExportRequest) {
     workspaceService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
         workspaceNamespace, workspaceId, WorkspaceAccessLevel.WRITER);
-    DataSetQueryList queryList =
-        generateQuery(workspaceNamespace, workspaceId, dataSetExportRequest.getDataSetRequest())
-            .getBody();
-    String queriesAsStrings =
-        "import pandas\n\n"
-            + queryList.getQueryList().stream()
-                .map(
-                    query ->
-                        convertQueryToString(
-                            query, dataSetExportRequest.getDataSetRequest().getName()))
-                .collect(Collectors.joining("\n\n"));
-    JSONObject notebookFile;
+    // This suppresses 'may not be initialized errors. We will always init to something else before
+    // used.
+    JSONObject notebookFile = new JSONObject();
     WorkspaceResponse workspace = fireCloudService.getWorkspace(workspaceNamespace, workspaceId);
+    JSONObject metaData = new JSONObject();
```

Comment:
I couldn't move it inside the if statement, because it was edited inside the scope of the else (And there's some general work that happens between the else and the if, even though its only used for new notebooks)

Reply:
I meant that to move the other code that uses `metaData` in there too.  Merging this commit will do it: 07a35a44109af4c8